pkg/utils/chart/mocks/gomock_reflect_608079190/prog.go

```diff
@@ -0,0 +1,64 @@
+package main
```

Comment:
Remove this file and also `pkg/utils/chart/mocks/gomock_reflect_608079190/prog.bin`